doc/neps/roadmap.rst

```diff
@@ -33,75 +39,76 @@ of the API, as discussed in `this section of NEP 37
 <https://numpy.org/neps/nep-0037-array-module.html#requesting-restricted-subsets-of-numpy-s-api>`__.
 
 
-Extensibility
--------------
-
-We aim to make it much easier to extend NumPy. The primary topic here is to
-improve the dtype system.
-
-- Easier custom dtypes:
-
-  - Simplify and/or wrap the current C-API
-  - More consistent support for dtype metadata
-  - Support for writing a dtype in Python
-
-- New string dtype(s):
-
-  - Encoded strings with fixed-width storage (utf8, latin1, ...) and/or
-  - Variable length strings (could share implementation with dtype=object,
-    but are explicitly type-checked)
-  - One of these should probably be the default for text data. The current
-    behavior on Python 3 is neither efficient nor user friendly.
-
-
 Performance
 -----------
 
 Improvements to NumPy's performance are important to many users. The primary
 topic at the moment is better use of SIMD instructions, also on platforms other
-than x86 - see :ref:`NEP38`.
+than x86 - see :ref:`NEP38`. The "universal SIMD" infrastructure is in place,
+and we welcome PRs to add SIMD support to NumPy functions.
```

Comment:
Improvements to NumPy's performance are important to many users.  We have
focused this effort on Universal SIMD (see :ref:`NEP38`) intrinsics which provide
nice improvements across various hardware platforms via an abstraction layer.
The infrastructure is in place, and we welcome follow-on PRs to add SIMD support
across all relevant NumPy functions.